libs/ec/inc/public/tfc/devices/abt/easycat.hpp

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include <bitset>
+#include <cstdio>
+
+#include "ethercat.h"
+#include "tfc/devices/base.hpp"
```

Comment:
in terms of the folderstructure should this perhaps be `tfc/ec/devices/base.hpp`